modules/tinymce/src/plugins/image/test/ts/browser/ContextMenuTest.ts

```diff
@@ -36,6 +50,25 @@ describe('browser.tinymce.plugins.image.ContextMenuTest', () => {
     TinyAssertions.assertSelection(editor, [], 0, [], 1);
   });
 
+  it('TINY-9491: Opening context menus on a selected figure', async () => {
```

Comment:
This test should be in the link plugin also it's more generic than the toc change.

So maybe just setup a div with noneditable links inside.

- [ ] Move test to `plugins/link/test/ts/browser/ContextMenuTest.ts`
- [ ] Make test more generic